phpstan-baseline.neon

```diff
@@ -1,11 +1,17 @@
 parameters:
-    ignoreErrors:
-        -
-            message: "#^Class Doctrine\\\\ODM\\\\PHPCR\\\\DocumentManager not found\\.$#"
-            count: 2
-            path: src/References/Mapping/Event/Adapter/ORM.php
+	ignoreErrors:
```

Comment:
Please, keep the previous indentation.

Reply:
That's how PHPStan dumps it.  The previous indention requires manual changes after `--generate-baseline`, and I'd say that's not a good practice for a machine-generated file.